include/FxMixer.h

```diff
@@ -69,6 +71,9 @@ class FxChannel : public ThreadableJob
 
 		bool requiresProcessing() const override { return true; }
 		void unmuteForSolo();
+		
+		QColor m_color;
+		bool m_hasColor;
```

Comment:
Sorry I didn't mention this sooner, but perhaps we could leave a note for future reference
```suggestion
		// TODO C++17 and above: Use std::optional instead
		QColor m_color;
		bool m_hasColor;
```

Reply:
Alright. Could I use something similar in the track color PR too?

Reply:
Yeah, I think that would be a good idea.